src/main/resources/org/jenkinsci/plugins/configfiles/ConfigFilesManagement/selectprovider.properties

```diff
@@ -18,4 +18,4 @@
 
 selectprovider=Select the file type you want to create
 filetype=Type
-
+configIdSuffixDescription=Config file ID is prefixed by the provider name (e.g. "org.jenkinsci.plugins.configfiles.maven.MavenSettingsConfig") and suffixed by the given "ID suffix" or by a random number if no "ID Suffix" is specified

```

Comment:
Wacky. Yeah that would have to be changed to a method `ConfigProvider.isResponsibleFor(Config)` so we would look up the actual config and do a typecheck or whatever on it.